ScoutSuite/cli_parser.py

```diff
@@ -1,264 +1,339 @@
+        azure_auth_params.add_argument('--username',
                                        action='store',
                                        default=None,
-                                       dest='azure_username',
-                                       help='Username of the Azure account')
-        azure_auth_params.add_argument('--azure-password',
+                                       dest='username',
+                                       help='Username of the Scout Suite account')
+        azure_auth_params.add_argument('--password',
                                        action='store',
                                        default=None,
-                                       dest='azure_password',
-                                       help='Password of the Azure account')
+                                       dest='password',
+                                       help='Password of the Scout Suite account')
+
+    def _init_common_args_parser(self):
+        parser = self.common_providers_args_parser.add_argument_group('ScoutSuite Arguments')
+
+        parser.add_argument('-l', '--local',
+                            dest='fetch_local',
+                            default=False,
+                            action='store_true',
+                            help='Use local data previously fetched and re-run the analysis.')
+        parser.add_argument('--resume',
+                            dest='resume',
+                            default=False,
+                            action='store_true',
+                            help='Complete a partial (throttled) run')
+        parser.add_argument('--update',
+                            dest='update',
+                            default=False,
+                            action='store_true',
+                            help='Reload all the existing data and only overwrite data in scope for this run')
+        parser.add_argument('--ruleset',
+                            dest='ruleset',
+                            default='default.json',
+                            nargs='?',
+                            help='Set of rules to be used during the analysis.')
+        parser.add_argument('--no-browser',
+                            dest='no_browser',
+                            default=False,
+                            action='store_true',
+                            help='Do not automatically open the report in the browser.')
+        parser.add_argument('--thread-config',
+                            dest='thread_config',
+                            type=int,
+                            default=4,
+                            help='Level of multi-threading wanted [1-5]; defaults to 4.')
+        parser.add_argument('--report-dir',
+                            dest='report_dir',
+                            default=DEFAULT_REPORT_DIR,
+                            help='Path of the ScoutSuite report.')
```

Comment:
Some places say `ScoutSuite`, and some other places say `Scout Suite`. We should choose one and be consistent. The Readme uses `Scout Suite`

Reply:
Yeah I use `Scout Suite` and `Scout` for short.

Reply:
Replaced everything with Scout